src/main/java/appeng/parts/misc/PartStorageBus.java

```diff
@@ -391,58 +427,54 @@ public MEInventoryHandler getInternalHandler()
 		this.monitor = null;
 		if( target != null )
 		{
-			final IExternalStorageHandler esh = AEApi.instance().registries().externalStorage().getHandler( target, this.getSide().getFacing().getOpposite(), StorageChannel.ITEMS, this.mySrc );
-			if( esh != null )
-			{
-				final IMEInventory inv = esh.getInventory( target, this.getSide().getFacing().getOpposite(), StorageChannel.ITEMS, this.mySrc );
+			IMEInventory<? extends IAEStack> inv = getInventoryWrapper( target );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'inv' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)